PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-java-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/java/sdk/utils/MavenUtils.java

```diff
@@ -101,4 +104,24 @@ public static String getLatestArtifactVersion(String groupId, String artifactId)
             return mavenArtifactDetails.getVersion();
         }
     }
+
+    /**
+     * This method checks if the method call is an Azure client method call
+     * by checking the containing class of the method call
+     *
+     * @param methodCall the method call to check
+     * @return boolean true if the method call is an Azure client method call, false otherwise
+     */
+    public static boolean isAzureClientMethodCall(PsiMethodCallExpression methodCall) {
```

Comment:
This method is unrelated to Maven. So, it doesn't belong in MavenUtils class. We should consider moving this to AzureUtils or something.